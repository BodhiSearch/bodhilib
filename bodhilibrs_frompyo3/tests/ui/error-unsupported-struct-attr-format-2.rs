use frompyo3::FromPyO3;

#[derive(FromPyO3)]
#[frompyo3(name Resource)]
struct TestUnsupportedStructAttr {}

fn main() {}
