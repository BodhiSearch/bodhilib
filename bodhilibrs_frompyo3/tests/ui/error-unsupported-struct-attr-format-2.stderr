error: Unsupported struct attribute format
 --> tests/ui/error-unsupported-struct-attr-format-2.rs:4:1
  |
4 | #[frompyo3(name Resource)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^
